.env-regtest-sample

```diff
@@ -1,53 +1,34 @@
 ###############################################################
-# RegTest Sparkswap Environment File
-#
-# This file contains settings that should be reviewed (and changed) when
-# running your broker daemon in a particular environment.
+# Sparkswap Environment File
 #
 # These settings control the underlying functionality of sparkswapd and the
 # associated lightning implementations (engines).
 #
 ###############################################################
 
 # Set the network that sparkswap and engines will work on.
-# Valid options are simnet, regtest, and testnet.
-#
 NETWORK=regtest
 
-# RPC User and Password for accessing the Sparkswap Daemon. If you are accessing
-# the daemon from outside of a closed network, you should update these values
-# for your security.
-RPC_USER=sparkswap
```

Comment:
Why remove the the `RPC_USER` and `RPC_PASS` from this file?

Reply:
@treygriffith I removed these because they are set as defaults, however it is probably good to add them back in the sample for consistency